github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-periodics.yaml

```diff
@@ -113,3 +113,61 @@ periodics:
     - name: gcs
       secret:
         secretName: gcs
+- name: periodic-containerized-data-importer-push-nightly-ARM64
+  cron: "2 3 * * *"
+  decorate: true
+  annotations:
+    testgrid-create-test-group: "false"
+  decoration_config:
+    timeout: 1h
+    grace_period: 5m
+  max_concurrency: 1
+  labels:
+    preset-dind-enabled: "true"
+    preset-docker-mirror-proxy: "true"
+    preset-shared-images: "true"
```

Comment:
This preset only makes sense for windows or rhel tests.
```suggestion
```

Reply:
Okay fixed